README.md

```diff
@@ -84,7 +84,7 @@ For additional instructions, standards and style guides, please refer to the [Co
 
 ## Careers
 
-See our open positions on [Greenhouse](https://boards.eu.greenhouse.io/evmos).
+See our open positions on [Personio](https://altiplanic.jobs.personio.de/?language=en).
```

Comment:
maybe would be even nicer to link to the careers page on evmos.org:

```suggestion
See our open positions on [our Careers page](https://evmos.org/careers/).
```

Reply:
Makes sense!